docs/manifest.json

```diff
@@ -197,6 +202,7 @@
             "name": "google/cloud-spanner",
             "defaultService": "spanner/spannerclient",
             "versions": [
+                "v0.5.1",
```

Comment:
https://github.com/GoogleCloudPlatform/google-cloud-php/pull/658/files#diff-02ab695754e489de3864df6c9010495eR31

I think this may be a breaking change, what do you think about bumping the minor?